data/info.json

```diff
@@ -191,5 +191,208 @@
         ]
       }
     ]
-  }
-}
\ No newline at end of file
+  },
+ "Alameda County": {
+   "info": [
+   {
+	   "header": "Testing Locations",
+	   "date": "2021-03-24",
+	   "body": [
+		    {
+			"h4": "COVID-19 Testing Sites in Alameda County",
+			"content": [
+				    {
+					"p2": [
+						{
+						"p": "Alameda County residents, including those who don’t have insurance, can contact the testing sites listed at the link below. You will not be asked immigration status and you are not required to give insurance information."
+						}
+					      ],
+					        "link": "http://www.acphd.org/2019-ncov/testing.aspx",
```

Comment:
after this "link" should be a closing bracket. The p2 below starts a new bracket.

```},{```

Reply:
<img width="645" alt="Screen Shot 2021-03-25 at 10 37 54 AM" src="https://user-images.githubusercontent.com/5452152/112517980-2f961180-8d56-11eb-969a-22ba2af2ac17.png">